.konflux/main/tests.yaml

```diff
@@ -12,8 +12,6 @@ spec:
   params:
     - name: POLICY_CONFIGURATION
       value: tekton-ecosystem-tenant/tekton-ecosystem-tenant-containers
-    - name: TIMEOUT
```

Comment:
We need to handle this in `hack`

Reply:
For all the components its working with default timeout onlu UI required this much time but it won't harm if we configure
so i think i can add changes to hack

Reply:
👍🏼

Reply:
Added https://github.com/openshift-pipelines/hack/commit/3812a0bbc8913d8f1c5876a73b956d8986c201ba